src/download.rs

```diff
@@ -355,7 +358,9 @@ impl SnapshotDownloader {
             .await
             .context(error::WriteFileBytes { path, count })?;
 
-        f.flush().await.context(error::FlushFile { path })?;
+        if !DISABLE_SHA {
```

Comment:
The formatting looks a bit off here - can we make sure to run all the code through `rustfmt`?